src/Selenium2Driver.php

```diff
@@ -240,27 +205,27 @@ protected static function charToOptions($char, $modifier = null)
             $ord = $char;
         }
 
-        $options = array(
-            'keyCode'  => $ord,
-            'charCode' => $ord
-        );
+        $options = [
+            'keyCode' => $ord,
+            'charCode' => $ord,
+        ];
 
         if ($modifier) {
-            $options[$modifier.'Key'] = 1;
+            $options[$modifier . 'Key'] = 1;
         }
 
         return json_encode($options);
     }
 
     /**
-     * Executes JS on a given element - pass in a js script string and {{ELEMENT}} will
```

Comment:
Of course it will show as JavaScript error in IDE because `{{ELEMENT}}` is invalid syntax. Since it works please replace `__ELEMENT__` back into `{{ELEMENT}}`.